.github/workflows/deploy.yml

```diff
@@ -76,9 +88,14 @@ jobs:
               cd /home/azureuser/notaries
               docker compose -f docker-compose.yml down || true
```

Comment:
If I understand correctly, adding a version will stop and restart other versions. 
Isn't it good enough to just run `docker compose up` to apply the changes?

Reply:
@heeckhau correct. we need to do docker-compose down because we recreate the directories (container mounts). Our next step is to migrate everything to docker swarm. Can we defer removing the compose down step in a new PR covering the swarm?